src/screens/Home/Home.tsx

```diff
@@ -83,14 +84,12 @@ export const Home: React.FC<undefined> = () => {
   }, 1500);
 
   function searchRestaurants(text: string) {
-    {
-      if (text.length > 1) {
-        setRestaurants([]);
-        setFilter({ text: text, page: 0 });
-      } else if (text.length <= 1) {
-        setRestaurants([]);
-        setFilter({ text: '', page: 0 });
-      }
+    if (text.length > 1) {
+      setRestaurants([]);
+      setFilter({ text: text, page: 0 });
+    } else if (text.length <= 1) {
+      setRestaurants([]);
+      setFilter({ text: '', page: 0 });
```

Comment:
se no else if nao for condicao contraria pode colocar `} else { ` pois nao esta colocando condicao diferente da mesma